docs/misc/release-notes.md

```diff
@@ -59,6 +59,16 @@ issues. You might have to flush your cookies if it doesn't work right away.
 For more information, see the [Docker forum
 post](https://forums.docker.com/t/new-safari-in-yosemite-issue/300).
 
+### Deprecated Features
```

Comment:
LOL no it was just an example.  My number 3 even said we need to find the right example.  Any suggestions?  Thinking about recent PRs, what about dropping "docker inspect" w/o the "--type" flag?  Or drop using it for images w/o "--type image"?